pkg/extractor/extractor.go

```diff
@@ -54,7 +54,7 @@ func ExtractConfigurationsFromYamlFile(path string) (*[]Configuration, string, *
 
 	configurations, err := ParseYaml(content)
 	if err != nil {
-		return nil, "", &InvalidFile{Path: absolutePath, ValidationErrors: []error{&InvalidYamlError{ErrorMessage: err.Error()}}}
+		return nil, "", &InvalidFile{Path: absolutePath, ValidationErrors: []error{&InvalidYamlError{ErrorMessage: "file content is not valid yaml"}}}
```

Comment:
I understand you want to make the error human readable but this will affect `datree test` output as well for files that are invalid yaml files
I think we should consider using the same error that is returned from `ParseYaml` to avoid masking the actual place the yaml failed.
If you want to change the output of the error specifically for the `validate-yaml` command then this should be located somewhere specific to that command (maybe inside its `main.go` file?

Reply:
I'll remove for now